>From 86b75aa04d6d35a93d76a023aa794a82488e791d Mon Sep 17 00:00:00 2001
From: Antonio Ospite <ospite@studenti.unina.it>
Date: Thu, 7 Mar 2013 12:50:25 +0100
Subject: [PATCH] debian/patches/: #0005 updated to avoid a crash
X-Face: z*RaLf`X<@C75u6Ig9}{oW$H;1_\2t5)({*|jhM<pyWR#k60!#=#>/Vb;]yA5<GWI5`6u&+
 ;6b'@y|8w"wB;4/e!7wYYrcqdJFY,~%Gk_4]cq$Ei/7<j&N3ah(m`ku?pX.&+~:_/wC~dwn^)MizBG
 !pE^+iDQQ1yC6^,)YDKkxDd!T>\I~93>J<_`<4)A{':UrE

Fix fill_locales() which is still looking for the locales dir in
BLENDER_SYSTEM_DATAFILES.

Closes: #702503
---
 debian/patches/0005-locales_directory_install.patch |   13 +++++++++++--
 1 file changed, 11 insertions(+), 2 deletions(-)

diff --git a/debian/patches/0005-locales_directory_install.patch b/debian/patches/0005-locales_directory_install.patch
index fc07d59..288d855 100644
--- a/debian/patches/0005-locales_directory_install.patch
+++ b/debian/patches/0005-locales_directory_install.patch
@@ -4,16 +4,25 @@ Subject: locales_directory_install
 
 Change where locales are installed.
 ---
- source/blender/blenfont/intern/blf_lang.c | 2 +-
+ source/blender/blenfont/intern/blf_lang.c | 4 ++--
  source/blender/blenlib/BLI_path_util.h    | 1 +
  source/blender/blenlib/intern/path_util.c | 3 +++
  source/creator/CMakeLists.txt             | 4 ++--
- 4 files changed, 7 insertions(+), 3 deletions(-)
+ 4 files changed, 8 insertions(+), 4 deletions(-)
 
 diff --git a/source/blender/blenfont/intern/blf_lang.c b/source/blender/blenfont/intern/blf_lang.c
 index 73294f1..dc0ba58 100644
 --- a/source/blender/blenfont/intern/blf_lang.c
 +++ b/source/blender/blenfont/intern/blf_lang.c
+@@ -78,7 +78,7 @@ static void free_locales(void)
+ 
+ static void fill_locales(void)
+ {
+-	char *languages_path = BLI_get_folder(BLENDER_DATAFILES, "locale");
++	char *languages_path = BLI_get_folder(BLENDER_SYSTEM_LOCALE, NULL);
+ 	LinkNode *lines = NULL, *line;
+ 	char *str;
+ 	int idx = 0;
 @@ -185,7 +185,7 @@ EnumPropertyItem *BLF_RNA_lang_enum_properties(void)
  void BLF_lang_init(void)
  {
-- 
1.7.10.4

